src/js/editor/post.js

```diff
@@ -9,9 +11,24 @@ function isMarkupSection(section) {
   return section.type === MARKUP_SECTION_TYPE;
 }
 
+function isTextEditableSection(section) {
```

Comment:
Yeah, this is a little confusing, I agree. I was expecting this to deal with the other text-ish section types (list section, list item) so opted for a more semantic name (rather than the rigid `isMarkupSection`), but later refactorings changed how those are dealt with and this ended up not needing any complexity. I could change the other parts of the post editor to use isMarkupSection instead.